test/suites/container_devices_nic_ipvlan.sh

```diff
@@ -7,67 +7,72 @@ test_container_devices_nic_ipvlan() {
     return
   fi
 
-  ct_name="nt$$"
+  ctName="nt$$"
```

Comment:
Can you split that rename into its own commit so it's a bit easier to handle cherry-picks?